name-generator.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"math/rand"
+	"time"
+)
+
+var (
+    left = [...]string{
+        "admiring",
+        "adoring",
+        "affectionate",
+        "agitated",
+        "amazing",
+        "angry",
+        "awesome",
+        "beautiful",
+        "blissful",
+        "bold",
+        "boring",
+        "brave",
+        "busy",
+        "charming",
+        "clever",
+        "cool",
+        "compassionate",
+        "competent",
+        "condescending",
+        "confident",
+    }
+    right = [...]string{
+        "agnesi",
+        "albattani",
+        "allen",
+        "almeida",
+        "antonelli",
+        "archimedes",
+        "ardinghelli",
+        "aryabhata",
+        "austin",
+        "babbage",
+        "banach",
+        "banzai",
+        "bardeen",
+        "bartik",
+        "bassi",
+        "beaver",
+        "bell",
+        "benz",
+        "bhabha",
+        "black",
+    }
+)
+
+func NameGenerator() string {
+    source := rand.NewSource(time.Now().UnixNano())
+    generator := rand.New(source)
+    leftn := generator.Intn(20)
```

Comment:
don't hard code the length. you already can do `Intn(len(left))` this way you don't have to count how many words you have in the list